Userland/Applications/CrashReporter/main.cpp

```diff
@@ -277,14 +277,20 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto& save_backtrace_button = *widget->find_descendant_of_type_named<GUI::Button>("save_backtrace_button");
     save_backtrace_button.set_icon(TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/save.png"sv)));
     save_backtrace_button.on_click = [&](auto) {
+        auto const show_error = [&](auto&& error) {
+            GUI::MessageBox::show(window, DeprecatedString::formatted("Couldn't save file: {}.", error), "Saving backtrace failed"sv, GUI::MessageBox::Type::Error);
+        };
```

Comment:
Maybe we should keep those messages separate.

I'd be interested in whether we failed to communicate with the FileSystemAccessServer or whether we just failed to write to the file for whatever reason.